src/sgame/sg_bot_skilltree.cpp

```diff
@@ -160,14 +162,12 @@ static Util::optional<botSkillTreeElement_t> ChooseOneSkill(const gentity_t *bot
 std::pair<std::string, skillSet_t> BotDetermineSkills(gentity_t *bot, int skill)
 {
 	std::vector<botSkillTreeElement_t> possible_choices = initial_unlockable_skills;
-	// aliens have 51 points to spend max
+	// aliens have 61 points to spend max
 	// humans have 40 points to spend max
-	// here we give a bit more money to humans because they have more
-	// expensive skills
-	float max = G_Team(bot) == TEAM_ALIENS ? 51.0f : 46.0f;
+	float max = G_Team(bot) == TEAM_ALIENS ? 61.0f : 40.0f;
```

Comment:
Why did the amount of human skill points decrease?

Reply:
It's actually about the same, since 40/7 ≃ 46/8.

Reply:
it's also exactly the same for skill5, since (int)(46.0f÷8×5) = (int)28.75 = 28 and (int)(40÷7×5) = (int)28.57 = 28

Reply:
I see. #2325 things :stuck_out_tongue: